context/app/static/js/components/organ/Samples/Samples.tsx

```diff
@@ -1,64 +1,25 @@
 /* eslint-disable no-underscore-dangle */
```

Comment:
We should be able to remove this eslint-disable - `_id` and `_source` are global exceptions to this rule